backend/tracim_backend/tests/functional/test_contents.py

```diff
@@ -29,6 +29,26 @@ class TestFolder(FunctionalTest):
 
     fixtures = [BaseFixture]
 
+    def _setup_basics(self) -> None:
```

Comment:
this implementation is something like `setUp()` / `tearDown()` pattern, no?
The way to implement it is to define a `setup_class()` class method, https://docs.pytest.org/en/latest/xunit_setup.html

Reply:
You seem not to share the use of `_setup_basics()` in all your test cases... so probably you must implement a new FolderTest class (eg `TestFolderMove`)